packages/itwin/tree-widget/src/test/trees/stateless/models-tree/internal/VisibilityChangeEventListener.test.ts

```diff
@@ -0,0 +1,95 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { expect } from "chai";
+import sinon from "sinon";
+import { BeEvent, using } from "@itwin/core-bentley";
+import { createVisibilityChangeEventListener } from "../../../../../components/trees/stateless/models-tree/internal/VisibilityChangeEventListener";
+import { createFakeSinonViewport } from "../../../Common";
+
+import type { Viewport } from "@itwin/core-frontend";
+describe("VisibilityChangeEventListener", () => {
+  it("raises event on `onAlwaysDrawnChanged` event", async () => {
+    const evt = new BeEvent();
+    const vpMock = createFakeSinonViewport({ onAlwaysDrawnChanged: evt });
```

Comment:
I think you could just call `createFakeSinonViewport` without the event and then just use `vpMock.onAlwaysDrawnChanged.raiseEvent` to trigger it.